packages/process/src/node/raw-process.ts

```diff
@@ -69,10 +81,17 @@ export class RawProcess extends Process {
            normalize the error handling by calling the error handler
            instead.  */
         try {
-            this.process = spawn(
-                options.command,
-                options.args,
-                options.options);
+            if (options.type === 'fork') {
+                this.process = fork(
+                    options.command,
```

Comment:
Is not it confusing that `command` means a path to node module? How to define separate types for `fork` options with proper options and names, i.e.
```ts
export types RawProcessOptions = SpawnRawProcessOptions | ForkRawProcessOptions;
export type SpawnRawProcessOptions =  ProcessOptions;
export interface ForkRawProcessOptions {
    type: 'fork'
    module: string
    // and so on
}
```

Reply:
As `fork()` and `spawn()` have exactly the same signature, it's annoying the type can't be determined based on different interfaces.

Reply:
Updated PR to implement different option types for forking or spawning. The interface used to specify the options determines whether to fork or spawn.